client/app/components/dashboards/dashboard-grid.less

```diff
@@ -89,6 +89,7 @@
     > .visualization-renderer-wrapper {
       flex-grow: 1;
       position: relative;
+      height: 0;
```

Comment:
Please try just this fix - for me it worked without others (also, I tried in Chrome a minute ago - it was fine too). If it works - let's not touch other styles.

Reply:
Only works in connection with this other change for me: https://github.com/getredash/redash/pull/5236/files#diff-4e9dd1634afb5937186e1247809856826568d835e2167e33be776e08ae6ffc94L6

Everything else we could get rid of though